android/Gutenberg/src/main/java/org/wordpress/gutenberg/GutenbergView.kt

```diff
@@ -555,6 +555,11 @@ object GutenbergWebViewPool {
     fun getPreloadedWebView(context: Context): GutenbergView {
         if (preloadedWebView == null) {
             preloadedWebView = createAndPreloadWebView(context)
+        } else {
+            val parent = preloadedWebView!!.parent
+            if (parent != null) {
+                (parent as? android.view.ViewGroup)?.removeView(preloadedWebView!!)
+            }
         }
```

Comment:
```suggestion
        val currentView = preloadedWebView
        if (currentView == null) {
            preloadedWebView = createAndPreloadWebView(context)
        } else {
            (currentView.parent as? android.view.ViewGroup)?.removeView(currentView)
        }
```

Just a bit more Kotlin way. So, we remove the `!!` operators and also simplify the conditionals.
(Note this has been written as pseudocode)

Reply:
Thanks for the guidance! Addressed in https://github.com/wordpress-mobile/GutenbergKit/pull/152/commits/b4d3438498fad3de77120f814e4770471982ac03.